src/main/resources/animal-attribute-data.yml

```diff
@@ -32,13 +52,25 @@ animal-attribute-data:
       attributeName: SEX
       attributeValue: SHE
 
+    - animalType: CAT
+      attributeName: AGE
+      attributeValue: YOUNG <1 YEAR
+
+    - animalType: CAT
+      attributeName: AGE
+      attributeValue: ADULT 1-8 YEARS
+
+    - animalType: CAT
+      attributeName: AGE
+      attributeValue: SENIOR >8 YEARS
+
```

Comment:
Tak samo jak dla psów. Plus imo niekoniecznie każdy kot i każda rasa psa po 8 latach to już senior

Reply:
rozni sie ale mniej wiecej gdzies po 8 jest senior wydaje mi sie, ewentualnie dac na 10 lat